docs/simulation/calculations.yaml

```diff
@@ -92,4 +92,32 @@ POPs:
               owners: total size of owner POPs in state
               workers: number of employed workers in RGO
               max(0.5: Hardcoded minimum of half the RGO income.
-              2*: Hardcoded value.
\ No newline at end of file
+              2*: Hardcoded value.
+Military:
+  Reinforcements per army:
+    formula: 5 * round(supply factor * sum of unit reinforcements * REINFORCE_SPEED * local reinforce rate * (1 + reinforce_rate) * (1 + reinforce_speed) / 5)
+    supply factor:
+      if unit received all its supplies: 1
+      else: complex graph, see https://www.desmos.com/calculator/gu1zpo01qz
+    unit reinforcements: 1000 * max_strength * unit reinforce rate
+      max_strength: as defined in /units/<unit>.txt
+      1000: (max_)strength is defined in thousands of soldiers. Reinforcements per army uses the number of soldiers.
+      unit reinforce rate:
+        if unit is mobilised: 0.5
+        else if pop is too small for a single unit: 0.5
```

Comment:
Is this if the pop is smaller than `defines.military.POP_MIN_SIZE_FOR_REGIMENT = 1000`? I believe this is also modified by:
- `defines.military.POP_MIN_SIZE_FOR_REGIMENT_PROTECTORATE_MULTIPLIER = 8`
- `defines.military.POP_MIN_SIZE_FOR_REGIMENT_COLONY_MULTIPLIER = 5`
- `defines.military.POP_MIN_SIZE_FOR_REGIMENT_NONCORE_MULTIPLIER = 3`

Reply:
I'd rather have this in a separate section.